standard/lpdb/condition.lua

```diff
@@ -13,21 +13,28 @@ local Array = require('Module:Array')
 local Condition = {}
 
 -- Abstract class, node of the conditions tree
+---@class _ConditionNode
+---@field is_a function
 local _ConditionNode = Class.new()
 
+---@enum lpdbBooleanOperator
```

Comment:
`---@enum` should be done directly on a table, so in this case `local BooleanOperator = {`